### PR TITLE
fix: Add retries and increase time between retries

### PR DESCRIPTION
**Pull request by ddubuc**

### CODE REVIEW

File: Client/src/Common/Submitter/BaseClientSubmitter.cs [hunk 1]
```diff
@@ -714,12 +738,16 @@ public ICollection<GetResultIdsResponse.Types.MapTaskResult> GetResultIds(IEnume
                         };
 
     var resultReply = Retry.WhileException(5,
-                                           200,
+                                           2000,
```
Comment (aneojgurhem) [inline Client/src/Common/Submitter/BaseClientSubmitter.cs:741]: We need to parametrize this value

Comment (aneojgurhem) [reply Client/src/Common/Submitter/BaseClientSubmitter.cs:741]: Could you add the issue please ?

Comment (ddubuc) [reply Client/src/Common/Submitter/BaseClientSubmitter.cs]: I was on it before your comment :)

File: Common/src/Common/RetryAction.cs [hunk 2]
```diff
@@ -142,4 +143,116 @@ public static class Retry
...
+  /// </summary>
+  /// <param name="retries">The number of times to retry the operation</param>
+  /// <param name="delayMs">The number of milliseconds to sleep after a failed invocation of the operation</param>
+  /// <param name="operation">the operation to perform</param>
+  /// <param name="exceptionType">if not null, ignore any exceptions of this type and subtypes</param>
+  /// <param name="allowDerivedExceptions">
+  ///   If true, exceptions deriving from the specified exception type are ignored as
+  ///   well. Defaults to False
+  /// </param>
+  /// <returns>When one of the retries succeeds, return the value the operation returned. If not, an exception is thrown.</returns>
```
[the remark this answers is not in the document]
Comment (lemaitre-aneo) [reply Common/src/Common/RetryAction.cs]: removed

**Pull request merged**